paws.common/R/credential_providers.R

```diff
@@ -361,18 +406,6 @@ iam_credentials_provider <- function() {
   return(creds)
 }
 
-# Get the name of the IAM role from the instance metadata.
```

Comment:
Moved `get_iam_role()` to `config.R` given parallel to `get_instance_metadata()` and similar functions, which pull from env variables